sys/vane/clay.hoon

```diff
@@ -3457,6 +3567,23 @@
       $boat
     :_  ..^$
     [hen %give %hill (turn ~(tap by mon.ruf) head)]~
+  ::.
+      $cred
+    =.  cez.ruf
+      ?~  cew.q.hic  (~(del by cez.ruf) nom.q.hic)
```

Comment:
`q.hic` could really use an alias. Comparing `p.q.hic` and `p.q.q.hic` a little later in this function is cruel and unusual punishment.

Reply:
You're right, even just aliasing `q.hic` to `req` makes a world of difference.
I also made a commit that adds actual faces to the `++task`. Not sure if that's boyscouting a bit too aggressively.